pages/events/_event.vue

```diff
@@ -33,7 +33,7 @@
     <!-- info section -->
     <v-container grid-list-xl>
       <h1 class="text-xs-center ma-3 primary--text font-lato">{{currentEvent.title}}</h1>
-      <v-layout row wrap justify-center>
+      <v-layout row wrap class="justify-center"r>
```

Comment:
remove extra `r` from after closing quotes
```suggestion
      <v-layout row wrap class="justify-center">
```